templates/gax/python/setup.py.mustache

```diff
@@ -22,7 +22,7 @@ setup(
     author_email='{{api.email}}',
     classifiers=[
         'Intended Audience :: Developers',
-        'Development Status :: 4 - Beta',
+        'Development Status :: 3 - Alpha',
```

Comment:
Downgrade to Alpha so that we can have more room to make backward-incompatible changes, right?